app/views/providers/search.html.erb

```diff
@@ -0,0 +1,3 @@
+<% ProviderSuggestion.suggest(params[:query]).each do |suggestion| %>
```

Comment:
This logic belongs in the controller and should be exposed via an instance variable